### DISCUSSION TITLE
show 47 counties at homepage

### DISCUSSION THREAD

**Discussion by 1969bahati** [lsc-hubs/kenya-hub#25] [General]
the 47 counties of Kenya needs to appear at the homepage. Once you click on the county, you need to get info on the county and subcounty as ward.

**Comment by pvgenuchten** [comment]
Thank you @1969bahati for this idea, what type of info would you like to see at your county page?
- the LSC participants in your county?
- the agricultural production/trade in your county?
- mereorological information about the area?
- the contact info of the local kalro office?
- ...

**Comment by pvgenuchten** [comment]
<https://selector.kalro.org> could be an interesting app for you
